azure/iac/master-compliance-test.json

```diff
@@ -2656,7 +2656,7 @@
                             "id": "PR-AZR-0114-ARM",
                             "eval": "data.rule.keySource",
                             "message": "data.rule.keySource_err",
-                            "remediationDescription": "",
+                            "remediationDescription": "Make sure StorageAccount has (properties.encryption.keySource='Microsoft.Keyvault')",
```

Comment:
**See my validation message at line 2496 (for id PR-AZR-0109-ARM).** make the description format like this one.

Reply:
fixed